css-module-bikeshed/Overview.bs

```diff
@@ -159,3 +159,7 @@ Internal display model: the 'foo' property {#the-foo-property}
 			DOMString baz = "qux";
 		};
 	</pre>
+
+<h2 class=no-num id=priv-sec>Privacy and Security Considerations</h2>
+
+Make some considerations about privacy and security.
```

Comment:
How about making this “Make some considerations about privacy and security.” into an issue, to make sure it doesn't get forgotten?